Resources/public/js/views/ez-serversideview.js

```diff
@@ -22,11 +22,41 @@ YUI.add('ez-serversideview', function (Y) {
     Y.eZ.ServerSideView = Y.Base.create('serverSideView', Y.eZ.View, [Y.eZ.Tabs, Y.eZ.SelectionTable], {
         events: {
             'form': {
-                'submit': '_submitForm'
+                'submit': '_confirmSubmit'
             },
         },
 
         /**
+         * Check if the submit action requires a confirm box,
+         *
+         * @method _checkConfirmBox
+         * @protected
+         * @param {EventFacade} e
+         */
+        _confirmSubmit: function (e) {
+            var focusedNode = e.target.get('ownerDocument').get('activeElement'),
+                details;
+
+            if (focusedNode.hasClass('ez-confirm-action')) {
+                if (this.get('container').one(focusedNode.getAttribute('data-confirmbox-details-selector'))) {
```

Comment:
this can be simplified by defining a local variable containing `this.get('container').one(focusedNode.getAttribute('data-confirmbox-details-selector'))`